.github/workflows/build-mudlet.yml

```
@@ -7,380 +7,427 @@ on:
-        fi
+      - name: Checkout Mudlet source code
+        uses: actions/checkout@v5
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v4
+        with:
+          version: ${{matrix.qt}}
+          dir: ${{runner.workspace}}
```

actionlint (1.7.8)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)